client/stylesheets/_toCat.scss

```diff
@@ -4987,7 +4996,8 @@ On click the images will be displayed at normal size to complete the effect
 }
 
 .notifMessage p span {
-    font-weight: 700 !important;
+    font-weight: 400 !important;
+    font-family: 'Helvetica Neue', sans-serif !important;
```

Comment:
!important should not be used<br>Line should be indented 2 spaces, but was indented 4 spaces